Ballerina.toml

```diff
@@ -0,0 +1,6 @@
+[project]
+org-name= "wso2"
```

Comment:
is this connector under wso2 org or ballerina org?

Reply:
we will host this under the org ballerinax